tests/apollo/util/bft.py

```diff
@@ -146,7 +146,8 @@ async def wrapper(*args, **kwargs):
                               f'with n={config.n}, f={config.f}, c={config.c}, '
                               f'num_clients={config.num_clients}, '
                               f'num_ro_replicas={config.num_ro_replicas}')
-                        await async_fn(*args, **kwargs, bft_network=bft_network)
+                        with log.start_task(action_type=async_fn.__name__):
```

Comment:
Is it also possible to log the parameters of the test, such as `n`, `f`, `c`, as well as `num_clients` ?

Reply:
Sure